docs/version_3.md

```diff
@@ -1,13 +1,40 @@
 # Version 3.0!
 
-We have been working hard the past few months on a number of features and improvements we wanted to combine into a major release. Though we intend that 3.0 to be backwards compatible, we thought it was a good idea to move to a new major version because we are changing a core part of how Style Dictionary works. 
+Version 3.0 is now publicly released! We have been working hard the past few months on a number of features and improvements we wanted to combine into a major release. Though we intend that 3.0 to be backwards compatible, we thought it was a good idea to move to a new major version because we are changing a core part of how Style Dictionary works. 
 
-If you would like to get a pre-release of 3.0, use the `next` tag to install Style Dictionary:
+If you are starting a new project, you can install Style Dictionary and it will give you the latest version:
 
-`npm install --save-dev style-dictionary@next`
+`npm install --save-dev style-dictionary`
+
+If you have an existing project, you can upgrade to 3.0 by updating the version in your `package.json` file to `"style-dictionary": "^3.0.0"` and then run `npm install` or you can use the `latest` tag to update both your `package.json` and `package-lock.json` files:
+
+`npm install --save-dev style-dictionary@latest`
 
 If you find any bugs or issues, please file tickets so we can get things fixed before we release it to the wider community. Thanks!
 
+## What's new in 3.0:
+
+* [Style Properties → Design Tokens](#style-properties-→-design-tokens)
+* [Transitive transforms](#transitive-transforms)
+* [Output references](#output-references)
+* [Custom parser support](#custom-parser-support)
+* [Adding filePath and isSource entries on tokens](#adding-filepath-and-issource-entries-on-tokens)
+* [Format helpers](#format-helpers)
+* [Updated format method arguments](#updated-format-method-arguments)
+* [Custom file headers](#custom-file-headers)
+* [Typescript support](#typescript-support)
+* [More built-ins](#more-built-ins)
+* [Bug fixes](#bug-fixes)
+* [Other features](#other-features)
+* [Better testing](#better-testing)
+* [Dropping support for older versions of node](#dropping-support-for-older-versions-of-node)
+
+### Style Properties → Design Tokens
+
+One last change is Style Dictionary is moving to the term "design tokens", both in documentation and in code. This has become the industry standard term for a while and it is time we respect that. Until now, Style Dictionary had called these "style properties" or just "properties", with some parts of the documentation also mentioning "design tokens". We want to be consistent with the direction of the community as well as in our documentation and code. We use the terms `properties` and `allProperties` in different APIs in Style Dictionary. To be consistent in documentation as well as code, we will be moving to using `tokens` and `allTokens`.
```

Comment:
```suggestion
Style Dictionary is moving to the term "design tokens", both in documentation and in code. This has become the industry standard term for a while and it is time we respect that. Until now, Style Dictionary had called these "style properties" or just "properties", with some parts of the documentation also mentioning "design tokens". We want to be consistent with the direction of the community as well as in our documentation and code. We use the terms `properties` and `allProperties` in different APIs in Style Dictionary. To be consistent in documentation as well as code, we will be moving to using `tokens` and `allTokens`.
```

Feels like off to start the first point in the list with "One last change is..." maybe?

Reply:
🤦 ah in my hastiness...

Reply:
My hastiness brought me here to see if V3 was out. 🤪